doc/lua_api.md

```diff
@@ -7888,17 +7792,23 @@ child will follow movement and rotation of that bone.
     object.
 * `set_detach()`: Detaches object. No-op if object was not attached.
 * `set_bone_position([bone, position, rotation])`
-	* Shorthand for `set_bone_override(bone, {position = position, rotation = rotation:apply(math.rad)})` using absolute values.
-	* **Note:** Rotation is in degrees, not radians.
-	* **Deprecated:** Use `set_bone_override` instead.
+    * Shorthand for `set_bone_override(bone, {position = position, rotation = rotation:apply(math.rad)})` using absolute values.
+> [!WARNING]
+> **Deprecated**, use `set_bone_override` instead.
+
+> [!NOTE]
+> Rotation is in degrees, not radians.
 * `get_bone_position(bone)`: returns the previously set position and rotation of the bone
-	* Shorthand for `get_bone_override(bone).position.vec, get_bone_override(bone).rotation.vec:apply(math.deg)`.
-	* **Note:** Returned rotation is in degrees, not radians.
-	* **Deprecated:** Use `get_bone_override` instead.
+    * Shorthand for `get_bone_override(bone).position.vec, get_bone_override(bone).rotation.vec:apply(math.deg)`.
+> [!WARNING]
+> **Deprecated**, use `get_bone_override` instead.
+
+> [!NOTE]
+> Returned rotation is in degrees, not radians.
 * `set_bone_override(bone, override)`
     * `bone`: string
-    * `override`: `{ position = property, rotation = property, scale = property }` or `nil`
-        * `property`: `{ vec = vector, interpolation = 0, absolute = false}` or `nil`;
+    * `override`: `position = property, rotation = property, scale = property}` or `nil`
```

Comment:
missing `{`
also it's more readable with spacing

Reply:
Added back the `{`, but I will not be putting the spacing back. This is one of the points of this PR